app/v1/fullsubtree_reconciler.go

```diff
@@ -34,7 +34,7 @@ func (c *controller) InitializeDirectories(ctx context.Context) error {
 	}
 
 	// check if all subdirs are tracked and up-to-date, else, persist them.
-	subdirs, err := c.c.GetChildren(ctx, c.baseDir)
+	subdirs, err := c.c.GetChildren(ctx, c.baseDir, nil)
```

Comment:
The new addition of options changes the function signature. While I really like that we now take options into use, having to add `nil` to the function when using default values is not great developer experience (devex). Let's use variadic parameters instead so the resulting call ends up as:

```golang
subdirs, err := c.c.GetChildren(ctx, c.baseDir)
```